docs/generators/typescript-axios.md

```diff
@@ -24,6 +24,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |sortParamsByRequiredFlag|Sort method arguments to place required parameters before optional parameters.| |true|
 |supportsES6|Generate code that conforms to ES6.| |false|
 |useSingleRequestParameter|Setting this property to true will generate functions with a single argument containing all API endpoint parameters instead of one argument per parameter.| |false|
+|withImportUrl|Setting this property to true adds imports from 'url' to the generated code for NodeJS support| |false|
```

Comment:
Sorry to bother again, I just checked my code, and there is 1 modification more I had to make re formData:

https://github.com/OpenAPITools/openapi-generator/blob/f27799c6851c1902f56257a209a0802d1ab3f920/modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache#L192

```diff
- localVarRequestOptions.headers = {...localVarHeaderParameter, ...headersFromBaseOptions, ...options.headers};
+ localVarRequestOptions.headers = {...localVarHeaderParameter, ...headersFromBaseOptions, ...localVarFormParams.getHeaders(), ...options.headers};
```

It seems otherwise it doesn't add the necessary length headers. And because the function it generates can take a stream as input, it might be difficult / inconvenient to set the length as a custom options.headers

Reply:
@arminrosu now added also `form-data` import and the `getHeaders` call if the `FormData` object at hand has the method (browser `FormData` does not have the function so this change should not break browser usage)